packages/gatekeeper/src/components/tests/login.test.tsx

```diff
@@ -21,7 +21,9 @@ describe('gatekeeper/OauthLogin', () => {
       providers: fixtures.providers
```

Comment:
Lets add a test that uses both types of OAuth flows to codify what the expected URL should be

Reply:
on it